doc/man3/SSL_SESSION_get_time.pod

```diff
@@ -36,6 +39,10 @@ in seconds.
 SSL_SESSION_set_timeout() sets the timeout value for session B<s> in seconds
 to B<tm>.
 
+SSL_SESSION_get_time_ex() and SSL_SESSION_set_time_ex() extended functions use
+the time_t datatype instead of long, as a potential workaround for the Y2038
+problem on 32 bit systems.
```

Comment:
I would say: `... use the time_t datatype instead of long, as a potential workaround for the Y2038
problem on systems with 64 bit time_t type.`

It would be also nice to add BUGS section and write there that long is 32 bit on many systems thus making the old functions not Y2038 safe.